mocks/auth.js

```diff
@@ -18,7 +21,7 @@ class FakeAuth {
 
   deleteUser() {
     mockDeleteUser(...arguments);
-    return Promise.resolve('👍');
+    return Promise.resolve('Success');
```

Comment:
lol don't like my emojis? 😂

Reply:
I love them! But my IDE got confused over the characters 😅
Feel free to revert this if you'd like